packages/common/lib/src/ld_context.dart

```diff
@@ -143,20 +150,30 @@ final class LDAttributesBuilder {
   /// in the LaunchDarkly model; any expressions in feature flags that reference
   /// an attribute with a null value will behave as if the attribute did not
   /// exist.
-  LDAttributesBuilder set(String name, LDValue value) {
-    _trySet(name, value);
-
+  LDAttributesBuilder setValue(String name, LDValue value,
```

Comment:
I have some hesitation in changing this, but I guess the upgrade will already be extensive.